### DISCUSSION TITLE
How to export/migrate from one immich instance to another

### DISCUSSION THREAD

**Discussion by alex3025** [immich-app/immich#12665] [Q&A]
I created a temporary immich instance separated from my main one to review a big photo collection that I had and I wanted to do it without touching my main library.

Now that I'm done reviewing, how can I export/migrate the library from the temporary instance to my main one?

EDIT: **I do not want to replace** my main library, I want to MERGE them

And I do not care about the temporary immich instance metadata (like faces), just exporting the photos/videos with their EXIF data will be enough.

**Comment by bo0tzz** [comment]
https://immich.app/docs/administration/backup-and-restore

**Reply by alex3025** [reply]
Hmm, I should have clarified it better, I do not want to replace my main library, I want to merge the twos

**Reply by bo0tzz** [reply]
Ah, there's no way to merge two instances. You'll have to upload the files, for example with the cli https://immich.app/docs/features/command-line-interface

**Reply by alex3025** [reply]
Yeah, but how do I export correctly the files (I do not want to lose the EXIF data) from the temporary instance to upload them on the main one?

**Reply by bo0tzz** [reply]
The files are accessible on the filesystem in UPLOAD_LOCATION

**Reply by alex3025** [reply]
And I could just copy them from there and they are complete with their metadata?

**Reply by bo0tzz** [reply]
Yes

**Comment by vrgpy** [comment]
Immich-go

https://github.com/simulot/immich-go?tab=readme-ov-file#from-immich-sub-command
